cards.rb

```diff
@@ -20,15 +18,11 @@ def to_s
     string
   end
 
-  def create_new_card(list)
-    new_card =  Cards.new
-    @current_list = list
-    @cards << new_card
-  end
+
 
   def add_checklist(string)
-    @checklist_item = string
-    @checklist << @checklist_item
+    checklist_item = string
```

Comment:
don't you think this line is redundant?
you just could leave `@checklist << string`, don't you?